src/com/palmergames/bukkit/towny/command/InviteCommand.java

```diff
@@ -55,22 +55,21 @@ public InviteCommand(Towny instance) {
 	@Override
 	public List<String> onTabComplete(CommandSender sender, Command command, String alias, String[] args) {
 		
-		if (args.length == 1) {
-			return NameUtil.filterByStart(inviteTabCompletes, args[0]);
-		}
-		
-		if (args.length == 2) {
-			switch (args[0].toLowerCase()) {
-				case "accept":
-				case "deny":
-					try {
-						return NameUtil.filterByStart(TownyUniverse.getInstance().getDataSource().getResident(sender.getName()).getReceivedInvites()
-							.stream()
-							.map(Invite::getSender)
-							.map(TownyInviteSender::getName)
-							.collect(Collectors.toList()), args[1]);
-					} catch (TownyException ignored) {}
-			}
+		switch (args.length) {
+			case 1:
+				return NameUtil.filterByStart(inviteTabCompletes, args[0]);
+			case 2:
+				switch (args[0].toLowerCase()) {
+					case "accept":
+					case "deny":
```

Comment:
One thing I forgot to mention last time around was that accept/deny are configurable in the config. I dunno if we want to handle that.

Reply:
We do